src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
```

Comment:
The property 'spring.application.name' is missing in the updated code. If this property is required for your application, please add it back.